sql/core/src/main/scala/org/apache/spark/sql/execution/command/AnalyzeColumnCommand.scala

```diff
@@ -71,9 +71,8 @@ case class AnalyzeColumnCommand(
 
   private def analyzeColumnInTempView(plan: LogicalPlan, sparkSession: SparkSession): Unit = {
```

Comment:
Note that `analyzeColumnInTempView` is called only if temp view (including global) exists: https://github.com/apache/spark/blob/d082ad0abfe0bc26760626ae0ecb415a8d508a1f/sql/core/src/main/scala/org/apache/spark/sql/execution/command/AnalyzeColumnCommand.scala#L45-L57